faqs/ko/glossary.md

```diff
@@ -0,0 +1,130 @@
+# 용어 해설집
+
+## A
+
+## B
+
+* **바이트(Bite)**: CDP의 청산을 일으키는 행위

```

Comment:
These subheadings need to follow the same syntax as the English version so we can link directly to the definitions:

```
# Glossary of Terms

## A

## B

##### Bite

- the action of triggering a CDPs liquidation
```